modules/tinymce/src/core/test/ts/browser/content/HTMLDataURLsTest.ts

```diff
@@ -0,0 +1,32 @@
+import { Assertions, Pipeline, Logger, NamedChain, Chain } from '@ephox/agar';
+import Theme from 'tinymce/themes/silver/Theme';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Editor as McEditor, ApiChains } from '@ephox/mcagar';
+
+UnitTest.asynctest('browser.tinymce.core.content.HTMLDataURLsTest', (success, failure) =>  {
+  Theme();
+
+  const content = '<p><a href="data:text/plain;base64,SGVsbG8sIHdvcmxkCg==">Click me</a></p>';
+
+  const getSettings = (hasDataUrls: boolean) => ({
+    base_url: '/project/tinymce/js/tinymce',
+    allow_html_data_urls: hasDataUrls
+  });
+
+  Pipeline.async({}, [
+    Logger.t('Editor should not allow data urls by default', Chain.asStep({}, [ NamedChain.asChain([
```

Comment:
Please place this on a new line to show it's a new block. Also `Logger` is the old module that's largely deprecated now, so you should be using `Log.stepsAsStep` for new tests. eg:
```
Log.stepsAsStep('TINY-5951', 'Editor should not allow data urls by default', Chain.asStep({}, [
  NamedChain.asChain([
    ...
   ])
])
```

Reply:
Actually, if you use `Log` you can use `Log.chainsAsStep`. eg:
```
Log.chainsAsStep('TINY-5951', 'Editor should not allow data urls by default', [
  NamedChain.asChain([
    ...
  ])
])
```